src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -126,5 +126,11 @@ public async Task AddSearchParameterStatusAsync(IReadOnlyCollection<string> sear
             // they will be fully enabled
             await UpdateSearchParameterStatusAsync(searchParamUris, SearchParameterStatus.Supported);
         }
+
+        internal async Task DeleteSearchParameterStatusAsync(string url)
```

Comment:
>internal [](start = 8, length = 8)

nit: it's a bit strange to see Delete as `internal` and Add as `public`.
Can they be synced?